sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -348,3 +348,12 @@ def test_fowlkes_mallows_score_properties():
 def test_mutual_info_score_positive_constant_label(labels_true, labels_pred):
     # non-regression test for #16355
     assert mutual_info_score(labels_true, labels_pred) >= 0
+
+
+def test_check_clustering_error():
+    # Test warning message for continuous values
+    noise = np.random.rand(500)
+    wavelength = np.linspace(0.01, 1, 500) * 1e-6
+
+    with pytest(UserWarning):
```

Comment:
```suggestion
    with pytest.warns(UserWarning):
```

Reply:
I made the change. Let me know if you I need to change anything else